file_repository/file_connexion.py

```diff
@@ -197,3 +197,19 @@ def move(self, oldfilepath, newfilepath, filename, new_filename=None):
         elif self.is_('filestore'):
             os.rename(os.path.join(oldfilepath, filename),
                       os.path.join(newfilepath, new_filename))
+
+    @open_and_close_connection
+    def delete(self, filepath, filename):
+        if not filepath: filepath = ''
+        if filename is False:
+            raise NoFileNameExcept("Filename is not defined")
```

Comment:
@bealdav , if filename is False, it would not: raise NoFileNameExcept("Filename is not defined at '%s'" % (filepath)